packages/ui/src/components/IdenticonBadge.tsx

```diff
@@ -3,18 +3,28 @@ import { useMemo } from 'react'
 import { styled } from '@mui/material/styles'
 import { AccountBadge } from '../types'
 import MultixIdenticon from './MultixIdenticon'
+import { ICON_SIZE_MEDIUM, ICON_SIZE_SMALL } from '../constants'
 
 interface Props {
   className?: string
   badge?: AccountBadge
   address?: string
+  sideBadge?: boolean
+  small?: boolean
```

Comment:
I would prefer to have iconSize props with the string to set the size in rem. Or to stick with other components and have a prop 
```
sizeVariant?: 'small' | 'medium' | 'large' 
```

Reply:
Also, Icon sizes are more situational rather than unified, so putting a strict value is not a fit for every situation

Reply:
done